packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -1,12 +1,9 @@
 const _ = require(`lodash`)
-const crypto = require(`crypto`)
 const stringify = require(`json-stringify-safe`)
+const deepMap = require(`deep-map`)
```

Comment:
```suggestion
const deepMap = require(`deep-map`)
```

Unsure where this came from. Linting says that it is unused. Should be good to remove.

Reply:
https://github.com/gatsbyjs/gatsby/pull/20239/commits/a82d4c45216038fded6ddd11a7ec00b8034eeae1 fixes this